main.go

```diff
@@ -160,13 +177,23 @@ func SlightEchoHandler(writer http.ResponseWriter, request *http.Request) {
 	writer.Write(responsePayload)
 }
 
-func runWorker(c *cli.Context, n *server.Node) error {
-	time.Sleep(10 * time.Second)
-	return errors.New("WORKER NOT IMPLEMENTED")
+func runWorker(c *cli.Context, n *server.Node, g peer.ID) error {
+	fmt.Println("Sleeping before run worker...")
+	time.Sleep(5 * time.Second)
+	fmt.Println("Running worker...")
+
+	conn, err := n.Vat.Connect(c.Context, peer.AddrInfo{ID: g}, casm.BasicCap{"lb/chan", "lb/chan/packed"})
```

Comment:
Is this line correct?

Reply:
It looks right.  Are you hitting an error?

(you can omit the `"lb/chan/packed" if you want.)

Reply:
I think I figured out the error. I was checking if err == nil instead of if err != nil down below :/